reviews/migrations/0013_is_commission_review_field.py

```diff
@@ -11,6 +11,7 @@ def update_is_commssion_review(apps, schema_editor):
         SUPERVISOR_STAGE = 0
         if review.stage == SUPERVISOR_STAGE:
             review.is_commission_review = False
+            review.update_go()
```

Comment:
In theory this should work... However, the production DB isn't as clean as we would like. 

I'd feel better if the code was something like this:
```
if review.go:
    review.stage = CLOSED
```

It's a lot simpler, and should be safer.